stock_orderpoint_manual_procurement/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Orderpoint Manual Procurement",
+    "summary": "Allows to create procurement orders from orderpoints instead "
+               "of relying only on the scheduler.",
+    "version": "10.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "depends": [
+        "stock",
+        "stock_orderpoint_uom",
```

Comment:
@lreficent Why depending on this ? I see the few lines in computing procure_recommended_quantity, but I don't think both should be linked automatically.

Reply:
@rousseldenis well, maybe you are right, there is no need to use both modules together. I'll take care of this.

Reply:
This has been attended